home-api-tests/pom.xml

```diff
@@ -401,5 +401,16 @@
             <artifactId>commons-text</artifactId>
             <version>${commons-text-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
```

Comment:
Isn't it redundant?

Reply:
You are right. This is already in lines 383-386.  But then I need to delete line  385 (<version>${assertj-version}</version>)